azureproject/settings.py

```diff
@@ -93,3 +93,67 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+if 'RUNNING_IN_PRODUCTION' in os.environ:
+    # Configure allowed host names that can be served and trusted origins for Azure Container Apps.
+    ALLOWED_HOSTS = ['.azurecontainerapps.io']
+    CSRF_TRUSTED_ORIGINS = ['https://*.azurecontainerapps.io']
+    DEBUG = False
+    DEBUG_PROPAGATE_EXCEPTIONS = True
+
+    # SECURITY WARNING: keep the secret key used in production secret!
+    # Use this py command to create secret 
+    # python -c 'import secrets; print(secrets.token_hex())'
+    SECRET_KEY = os.getenv('AZURE_SECRET_KEY')
+
+    STATICFILES_STORAGE = 'whitenoise.storage.CompressedStaticFilesStorage'
+
+    # Configure database connection for Azure PostgreSQL Flexible server instance.
+    # AZURE_POSTGRESQL_HOST is the full URL.
+    # AZURE_POSTGRESQL_USERNAME is just name without @server-name.
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': os.environ['AZURE_POSTGRESQL_DATABASE'],
+            'HOST': os.environ['AZURE_POSTGRESQL_HOST'],
+            'USER': os.environ['AZURE_POSTGRESQL_USERNAME'],
+            'PASSWORD': os.environ['AZURE_POSTGRESQL_PASSWORD'], 
```

Comment:
You may want to specify sslmode=required. We often find that's necessary when connecting to Azure Postgres databases. However, the Postgres add-on sets a POSTGRES_SSL env variable to indicate whether it should be on/off. This is how we deal with it:
https://github.com/Azure-Samples/azure-django-postgres-flexible-aca/blob/main/src/project/settings.py#L107C1-L108C39

And then we set it to require in the infra:
https://github.com/Azure-Samples/azure-django-postgres-flexible-aca/blob/5d30fcab0354c14453575a8ee0ba29c0dadc87e7/infra/web.bicep#L65

You may not need that complexity depending on what postgres options you're supporting.